spec/services/personal_care_needs/importer_spec.rb

```diff
@@ -42,7 +42,7 @@
     it 'appends an assessment answer with the default values' do
       importer.call
 
-      expect(profile.assessment_answers.first.as_json).to include(
+      expect(profile.assessment_answers.first.as_json.symbolize_keys).to include(
```

Comment:
`AssessmentAnswers` previously had a custom implementation of `as_json` which (incorrectly) returned all attributes with symbolized keys. The `ActiveModel::Model` implementation instead (correctly) returns stringified keys. Keep in mind that `as_json` is then rendered with `to_json` so the symbol/string distinction completely disappears anyway. I've added the `symbolize_keys` call here to avoid having to change each of the following attributes in the spec.